app/validators/expiry_date_validator.rb

```diff
@@ -0,0 +1,10 @@
+class ExpiryDateValidator < ActiveModel::Validator
+  def validate(record)
+    if record.expires_at && record.expires_at <= DateTime.current
+      record.errors[:expires_at] << 'date must be in the future'
+      false
+    else
+      true
+    end
+  end
+end
```

Comment:
Doesn't look like this is being tested anywhere. Personally I don't think we need to return true or false here.
I just did some work on this myself and I think we can refactor to something like 
```
return if record.expires_at && record.expires_at <= DateTime.current
record.errors[:expires_at] << 'blehblehbleh'
```

Refrence: http://www.rails-dev.com/custom-validators-in-ruby-on-rails-4

If this was being tested somewhere it would be really easy to tell if this refactor works. ( I think it does ).